mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go.erb

```diff
@@ -1763,7 +1894,7 @@ resource "google_bigquery_table" "test" {
     hive_partitioning_options {
       mode = "AUTO"
       source_uri_prefix = "gs://${google_storage_bucket.test.name}/"
-	  require_partition_filter = true
+	    require_partition_filter = true
```

Comment:
Nit: the spacing changes in this file still look incorrect, so I would recommend either leaving them as-is, or correcting them

Reply:
Made another attempt to fix